dpx/src/dpx_pdfdev.rs

```diff
@@ -103,113 +105,43 @@ pub type spt_t = i32;
 
 pub type TMatrix = euclid::Transform2D<f64, (), ()>;
 
-/*#[derive(Copy, Clone, Default)]
-#[repr(C)]
-/// Transform coordinate matrix
-pub struct TMatrix {
-    pub a: f64,
-    pub b: f64,
-    pub c: f64,
-    pub d: f64,
-    pub e: f64,
-    pub f: f64,
-}
-impl TMatrix {
-    /// Zero initialized transform matrix
-    pub const fn new() -> Self {
-        Self {
-            a: 0.,
-            b: 0.,
-            c: 0.,
-            d: 0.,
-            e: 0.,
-            f: 0.,
-        }
-    }
-    /// Identity transform matrix
-    pub const fn identity() -> Self {
-        Self {
-            a: 1.,
-            b: 0.,
-            c: 0.,
-            d: 1.,
-            e: 0.,
-            f: 0.,
-        }
-    }
-}*/
-#[derive(Copy, Clone, Default)]
-#[repr(C)]
-/// Represents rectangle and TeX bbox in document
-pub struct Rect {
-    /// Lower left coorditate of rectangle
-    pub ll: Coord,
-    /// Upper right coorditate of rectangle
-    pub ur: Coord,
+pub type Rect = euclid::Box2D<f64, ()>;
+
+pub trait Corner {
+    fn lower_left(&self) -> Point;
+    fn upper_right(&self) -> Point;
+    fn lower_right(&self) -> Point;
+    fn upper_left(&self) -> Point;
 }
-impl Rect {
-    /// Zero initialized rectangle
-    pub const fn zero() -> Self {
-        Self {
-            ll: Coord::new(0., 0.),
-            ur: Coord::new(0., 0.),
-        }
+impl Corner for Rect {
+    fn lower_left(&self) -> Point {
+        self.min
     }
-    /// Create new rectangle from lower left and upper right coorditate
-    pub const fn new(ll: (f64, f64), ur: (f64, f64)) -> Self {
-        Self {
-            ll: Coord::new(ll.0, ll.1),
-            ur: Coord::new(ur.0, ur.1),
-        }
-    }
-    pub fn width(&self) -> f64 {
-        self.ur.x - self.ll.x
-    }
-    pub fn height(&self) -> f64 {
-        self.ur.y - self.ll.y
-    }
-    pub fn lower_left(&self) -> Coord {
-        self.ll
-    }
-    pub fn upper_right(&self) -> Coord {
-        self.ur
-    }
-    pub fn lower_right(&self) -> Coord {
-        Coord::new(self.ur.x, self.ll.y)
+    fn upper_right(&self) -> Point {
+        self.max
     }
-    pub fn upper_left(&self) -> Coord {
-        Coord::new(self.ll.x, self.ur.y)
+    fn lower_right(&self) -> Point {
+        point2(self.max.x, self.min.y)
     }
-}
-impl From<(Coord, Coord)> for Rect {
-    fn from(c: (Coord, Coord)) -> Self {
-        Self {
-            ll: c.0,
-            ur: c.1,
-        }
+    fn upper_left(&self) -> Point {
+        point2(self.min.x, self.max.y)
     }
 }
 
-impl std::fmt::Display for Rect {
```

Comment:
Note: This PR switches out the fmt::Display implementation of Rect. It doesn't seem to be used anywhere though.